tests/testthat/test-init_filter_state.R

```diff
@@ -20,34 +20,28 @@ test_that("'extract_type' must be specified with 'input_dataname'", {
 })
```

Comment:
This code raises an error because `ADSL` is not defined. Any suggestions would be welcome.